Podverse/Podcast.swift

```diff
@@ -39,19 +39,17 @@ class Podcast: NSManagedObject {
     static func retrieveMediaRefsFromServer(episodeMediaUrl: String? = nil, podcastFeedUrl: String? = nil, onlySubscribed: Bool? = nil, completion: @escaping (_ mediaRefs:[MediaRef]?) -> Void) {
     }
     
-    static func retrievePodcastUIImage(item: PlayerHistoryItem? = nil, podcast: Podcast? = nil, downloadingEpisode: DownloadingEpisode? = nil, completion: @escaping (_ podcastImage: UIImage?) -> Void) {
+    static func retrievePodcastUIImage(podcastFeedUrl: String?, podcastImageUrl: String?, completion: @escaping (_ podcastImage: UIImage?) -> Void) {
```

Comment:
The only issue I can think of with that is sometimes we're retrieving podcast images for clips, and therefore the podcasts are not stored in the MOC.

I'll add that as an optimization though. The first conditional will look for managed object id, and if not then feedUrl, else use podcastImageUrl, else nil.